microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -649,13 +667,39 @@ See also:
 * [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 125.
-* This checkbox is present in Microsoft Edge Stable 124.
+* This checkbox is present in Microsoft Edge Canary 128.
+* This checkbox is present in Microsoft Edge Stable 125.
+
+
+<!-- ====================================================================== -->
+## Enable save and load trace with annotations in Performance Panel
+<!-- checkbox has no (?) link pointing to this anchor wording -->
+
+_new as of Microsoft Edge 125_
+
+See the above section, showing the newer label for the checkbox.
+
+
+<!-- ====================================================================== -->
+## Enable new Performance panel landing page which includes live metrics and field data
+<!-- checkbox has no (?) link pointing to this anchor wording -->
+
+_new as of Microsoft Edge 128_
+
+In the **Performance** tool, displays a main page section that shows live metrics and field data.
+<!-- todo: need real desc -->
```

Comment:
Enable new Performance panel landing page which includes live metrics and field data
Need real desc @captainbrosset

Reply:
```
In the **Performance** tool, adds a landing page that's displayed when no performance traces 
have been recorded yet.  The landing page displays performance metrics about the rendered 
page that are updated live, such as:
* **Largest Contentful Paint (LCP)**
* **Cumulative Layout Shift (CLS)**
* **Interaction to Next Paint (INP)**
```